README.md

```diff
@@ -149,6 +149,22 @@ If both these parameters are present, the node will also start service SSL traff
       "Config": { // configuration passed to the binary inside the container
         "api": "v1"
       }
+    },
+    "custom-service": {
+      "InternalPort": 8080,
+      "ExternalPort": 2000,
+      "NeedsKeys": false, // should pass the keys as a file; **never** set it to true, default false (optional)
+      "Executable": "/opt/orbs/service", // default (optional)
+      "SignerNetworkEnabled": false, // should be able communicate with the signer service, default false (optional)
```

Comment:
again, this implies you know who the signer is by name. so why do you need `NeedsKeys` - only the signer needs the keys by definition.. no?

And, instead of having a key for each class of services, does it make sense to have one array field named `serviceBinding: ["signer", "management-service"]` and thats it? if this is the terminology you end up using with docker then maybe thats more elegant?

Reply:
`NeedsKeys` will be renamed as `InjectNodePrivateKey`

`SignerNetworkEnabled` will be renamed as `AllowAccessToSigner`

`ServicesNetworkEnabled` will be renamed as `AllowAccessToServices`

I am in favor of boolean toggles instead of the list for the following reasons:
* toggles are easier to generate in management service
* toggles are easier to modify as a human

Reply:
* its nicer to keep separation between nouns and verbs. in this case, keys are naturally nouns. so both `InjectNodePrivateKey` and `allow...` stand out as they are read like directives and not configuration. this is a minor stylistic point.

my only real question here - we can discuss offline - is why we need `InjectNodePrivateKey` if boyar already has hard coded knowledge of who to give these keys to...